index.bs

```diff
@@ -1324,43 +1332,31 @@ The [=sandboxed file system root=]'s [=query permission steps=] are the followin
 
 </div>
 
-## The {{FileSystemDirectoryHandle/getSystemDirectory()}} method ## {#api-getsystemdirectory}
+## The {{getSandboxedFileSystem()}} method ## {#api-getsandboxedfilesystem}
 
 <xmp class=idl>
-enum SystemDirectoryType {
-  "sandbox"
-};
-
-dictionary GetSystemDirectoryOptions {
-  required SystemDirectoryType type;
-};
-
 [SecureContext]
-partial interface FileSystemDirectoryHandle {
-  static Promise<FileSystemDirectoryHandle> getSystemDirectory(GetSystemDirectoryOptions options);
+partial interface Window {
+  Promise<FileSystemDirectoryHandle> getSandboxedFileSystem();
 };
 </xmp>
 
+Advisement: In Chrome this functionality is currently exposed as `FileSystemDirectoryHandle.getSystemDirectory({type: "sandbox"})`.
```

Comment:
Add the milestone? I also like tossing in 🚧 markers, but I'm silly that way.

Reply:
Well, currently it's all milestones... Once we've implemented these changes I definitely want to go back and update this to mention milestones, just not sure what milestone to mention now.